python/ambassador/envoy/v3/v3listener.py

```diff
@@ -965,17 +969,21 @@ def finalize_http(self) -> None:
             self._filter_chains.append(filter_chain)
 
     def as_dict(self) -> dict:
-        odict = {
+        listener = {
```

Comment:
You took this rename, but further down in the method it is still refers to `odict`.

Reply:
Likewise in v2listener.py